setup.py

```diff
@@ -56,7 +56,7 @@
         "importlib-metadata ; python_version<'3.8'",
         "pydantic>=1.9.0,<2.0",
         "hexbytes>=0.2.2,<1.0.0",
-        "eth-utils==1.10.0",
+        "eth-utils>=1.10.0",
```

Comment:
might need to add `eth-hash[pysha3]` to `test` extras, and then make a note that downstream should also install it here

Reply:
Where does this get used?